.gitignore

```diff
@@ -8,3 +8,4 @@
 /*.scss
 /*.sass
 /*.less
+.DS_Store
```

Comment:
Could you remove this change from this PR? You should probably add this to your [global `.gitignore`](https://stackoverflow.com/a/7335487/1358948).

Reply:
Yeah of course I can remove it.

Reply:
Done!